BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -907,6 +991,7 @@ private void ConvertTriggerPush(Entity trigger)
 				trigger.ClassName = "trigger_jumppad";
 				trigger["launchtarget"] = targets.First().Name;
 				trigger["spawnflags"] = "1";
+				targets.First().ClassName = "info_target";
```

Comment:
This can be improved a bit:
```
var firstTarget = targets.FirstOrDefault();
if (firstTarget != null)
{
    trigger.ClassName = "trigger_jumppad";
    trigger["launchtarget"] = firstTarget.Name;
    trigger["spawnflags"] = "1";
    firstTarget.ClassName = "info_target";
}
```